auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/UmsClient.java

```diff
@@ -783,4 +787,29 @@ public String getIdentityProviderMetadataXml(String requestId, String accountId)
         }
         return assignee.build();
     }
+
+    /**
+     * Retrieves user sync state model from the UMS.
+     *
+     * @param requestId          the request ID for the request
+     * @param accountId          the account ID
+     * @param rightsChecksList   list of mapping from resources to lists of rights to check. Lists are used to
+     *                           preserve order.
+     * @return the user sync state model
+     */
+    public GetUserSyncStateModelResponse getUserSyncStateModel(
+            String requestId, String accountId, List<Pair<String, List<String>>> rightsChecksList) {
```

Comment:
`List<Pair<String, List<String>>>` is a bit hard to read, maybe turn this into
`List<RightsCheck>` (adding a new RightsCheck class)?

Reply:
We already have some UserManagementProto classes in the GrpcUmsClient. I'll just expose the `RightsCheck` class all the way out.